docs/internals/howto-release-django.txt

```diff
@@ -95,7 +101,7 @@ any time leading up to the actual release:
    ``django.contrib.auth.hashers.PBKDF2PasswordHasher`` by about 20%
    (pick a round number). Run the tests, and update the 3 failing
    hasher tests with the new values. Make sure this gets noted in the
-   release notes (see release notes on 1.6 for an example).
+   release notes (see the 1.8 release notes for an example).
```

Comment:
It seems to me this could even be done as soon as we branch off an alpha, right after we remove deprecated features.